Exercises/2-array.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
-const array = () => null;
+const array = (a = [], f) =>
+  (f = i => a[i], f.push = s => a.push(s), f.pop = () => a.pop(), f);
```

Comment:
Cryptic code. Rewrite as a block of code, for better reading. Because this function is not in function-programming style, it uses mixins and mutable arrays, so it is not good idea to write it in a functional-style.